providers/src/airflow/providers/amazon/aws/hooks/s3.py

```diff
@@ -1326,10 +1327,13 @@ def copy_object(
         :param source_version_id: Version ID of the source object (OPTIONAL)
         :param acl_policy: The string to specify the canned ACL policy for the
             object to be copied which is private by default.
+        :param meta_data_directive: Specifies whether the metadata is copied from the source object or replaced with metadata that's provided in the request.
```

Comment:
```suggestion
        :param meta_data_directive: Whether to `COPY` the metadata from the source object or `REPLACE` it with metadata that's provided in the request.
```


Non-blocking suggestion for here and the other docstring, since the parameter is an open-ended string input.  Feel free to ignore.